cpp/ql/src/Security/CWE/CWE-704/WcharCharConversion.qhelp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+  <p>This rule indicates a potentially incorrect cast from/to an ANSI string (<code>char *</code>) to/from a Unicode string (<code>wchar_t *</code>).</p>
```

Comment:
I don't think it's accurate to say that `char` is ANSI and `wchar_t` is Unicode. That's what they're called in Windows APIs, but other platforms use `char` for both 7-bit ASCII, 8-bit legacy charsets, and UTF-8 Unicode.

I suggest replacing "ANSI string" with "byte string" and "Unicode string" with "wide-character string". Also below.